packages/webapp/public/locales/en/translation.json

```diff
@@ -65,7 +65,7 @@
     "HOW_MUCH_IS_HARVESTED": "How much is being harvested?",
     "HR": "/hr",
     "MANAGE_CUSTOM_TASKS": "Manage custom tasks",
-    "NEED_MANAGEMENT_PLAN": "You’ll need an active or planned crop plan before you can schedule a harvest task or transplant task. Go to the crop catalogue to create a plan now.",
+    "NEED_MANAGEMENT_PLAN": "You'll need an active or planned crop plan before you can schedule a harvest task or transplant task. Go to the crop catalogue to create a plan now.",
```

Comment:
I assume it's no problem, but I'll point out that the more exotic apostrophe characters that were in the files have been replaced with the mundane standard keyboard single quote. May have been done automatically by editor software.